network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_cta_card_block_regular.html

```diff
@@ -0,0 +1,32 @@
+{% load l10n wagtailcore_tags wagtailimages_tags %}
+
+<div class="[&_.rich-text_a]:tw-underline [&_.rich-text_a]:tw-text-black
```

Comment:
I've tried to create a [variant plugin](https://tailwindcss.com/docs/plugins#static-variants) for this selector as we might need to use it often but it wouldn't work.

After adding

```js
    plugin(function({ addVariant }) {
      addVariant("rich-text-links", ["&_.rich-text_a"]);
    })
```
to our `tailwind.config.js` and changing the calls to `rich-text-links:tw-underline rich-text-links:tw-text-black`, the styles were no longer applied to links inside the cards.

This is a weird issue, since the syntax for the plugin seems to be correct, as we can see in this [Tailwind play environment](https://play.tailwindcss.com/eZmFo3HlUs).

Reply:
It seems like this is caused by a bug with Tailwind and prefixes: https://github.com/tailwindlabs/tailwindcss/issues/10125#issuecomment-1382724160

There is a workaround, which I have now implemented here :)